tests/cypress/integration/z-newfold-labs/wp-module-onboarding/2-intermediate-steps/0-get-started/TC_06-sitetype-secondary.cy.js

```diff
@@ -25,17 +25,34 @@ describe( 'Get Started Site Type Secondary', function () {
 	} );
 
 	it( 'Check to make sure sidebar opens, content is in place and close sidebar', () => {
-		CheckIntroPanel( '__get-started-site-secondary', 'Store Type' );
+		CheckIntroPanel( '__get-started-site-secondary', 'Site Type' );
 		CheckIllustrationPanel();
 		CheckInfoPanel();
 		CheckHelpPanelLinks();
 	} );
 
 	it( 'Check selected category is visible and selected', () => {
 		cy.get( '.category-scrolling-wrapper' ).should( 'be.visible' );
-		cy.get( '.category-scrolling-wrapper__type-text' ).should( 'contain', 'Business' );
+		cy.get( '.category-scrolling-wrapper__type-text' ).should(
+			'contain',
+			'Business'
```

Comment:
I don't see that we have selected business in our primary category step, is this step working just because `woocommerce` is being pre-installed?

Reply:
Whether we have WooCommerce or not, by default the first category if we do not select any category will be Business.

Reply:
Is it a good idea to assert `Business` in the commerce flow and ensure that the `<` `>` buttons do not exist?